src/System.Linq.Expressions/tests/Unary/UnaryConvertTests.cs

```diff
@@ -102,6 +102,18 @@ public static void ConvertArrayToIncompatibleTypeTest(bool useInterpreter)
             }
         }
 
+        [Fact]
+        public static void ToStringTest()
+        {
+            // NB: Unlike TypeAs, the output does not include the type we're converting to
+
+            var e1 = Expression.Convert(Expression.Parameter(typeof(object), "o"), typeof(int));
+            Assert.Equal("Convert(o)", e1.ToString());
```

Comment:
Sounds good, will make the change together with Power. My main concern is around people using ToString for quick and dirty asserts of an expected tree shape. The lack of an IEqualityComparer for structural expression equality (including scoping of variables and label matching) could have driven folks to use a "signature" of the tree instead for comparison purposes.

Likely a separate issue should be opened to consider such a comparer. I have one here at Bing and can push it out some day for consideration.